frontend/src/components/LetterResult.js

```diff
@@ -0,0 +1,12 @@
+const LetterResult = (props) => {
+  const letterStyle = props.isPresent ? "letter-exists" : "letter-wrong";
+  const letterId = "letter-" + props.position;
+
+  return (
+    <div className={letterStyle} id={letterId}>
```

Comment:
You may want to check out the [`classNames`](https://www.npmjs.com/package/classnames) library for this

Reply:
- [ ] Look into classNames library to refactor LetterResult workflow